packages/rancher-vsphere-csi/charts/README.md

```diff
@@ -71,3 +71,9 @@ More information on managing Secrets using kubectl [here](https://kubernetes.io/
 ## Migration
 
 The CSI migration feature is only available for vSphere 7.0 U1.
+Follow the steps [here](https://rancher.com/docs/rancher/v2.6/en/cluster-provisioning/rke-clusters/cloud-providers/vsphere/out-of-tree/vsphere-volume-migration/?#) for CSI Migration.
```

Comment:
hmm we are linking 2.6 docs here but 2.5 docs below. Should they be consistent?

Reply:
good catch, updating now